packages/language/src/language-server/lsp-codes.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+import { Severity } from "../language-server/types";
+import { SimplePLICode } from "../validation/pli-codes";
+
+export const lspCodes = {
```

Comment:
Minor: Exported constants are usually either CamelCase or SNAKE_CASE.
```suggestion
export const LspCodes = {
```